app/_src/gateway/reference/reserved-entity-names.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Reserved Entity Names
+content_type: reference
+badge: enterprise
+---
+
+The {{site.base_gateway}} maintains a list of reserved entity names which cannot be used when naming workspaces or other entities.
+
```

Comment:
Should all of these entities be singular or plural? Is the plural name reserved but the singular isn't?